### DISCUSSION TITLE
RuntimeError: Server with name pdlstorage01 is not defined

### DISCUSSION THREAD

**Discussion by crookedmosquito** [Taxel/PlexTraktSync#1800] [Support]
Tried setting this up and got this error:

         RuntimeError: Server with name pdlstorage01 is not defined                                                                                  
Error: Error running sync command: Server with name pdlstorage01 is not defined

Any assistance on what the issue is?

**Comment by glensc** [comment]
nobody is that bored that they will start ask you random questions, provide information first.
